### PR TITLE
AIRO-356: Tutorial Updates

### PR DESCRIPTION
**Pull request by at669**
- More explanation on location of windows, menu options
- Highlighted sections of screenshots in Part 1
- Minor clarifications, additional screenshots
- Moved the default target/targetPlacement objects to the "front" (on the Unity logo) such that the action happens "in front of the camera" as seen below:
    Screen Shot 2021-01-06 at 1 57 18 PM
- Removed references to Message Generation window, updated to Robot menu options & Browse window
- Removed instructions to set up RosConnect object, updated scripts & instructions to reference `ROSConnection.instance`
- Updated miscellaneous screenshots to reflect these changes

### CODE REVIEW

File: tutorials/pick_and_place/2_ros_tcp.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ To enable communication between Unity and ROS, a TCP endpoint running as a ROS n
...
 
-1. We will start with generating the MoveItMsg: RobotTrajectory. 
+2. We will start with generating the MoveItMsg: RobotTrajectory. This file describes the trajectory contents that will be used in the sent and received trajectory messages.
 
    Select `RosMessageGeneration -> Auto Generate Messages -> Single Message` from the menu.
 
    ![](img/2_single.png)
 
-   In the Message Auto Generation window click `Browse File...` and navigate to the MoveIt Msgs repository. Select `moveit_msgs/msg/RobotTrajectory.msg`, and then click `GENERATE!` 
+   In the Message Auto Generation window click `Browse File...` and navigate to the MoveIt Msgs directory in this cloned repository (`Unity-Robotics-Hub/tutorials/pick_and_place/ROS/src/moveit_msgs`). Select `moveit_msgs/msg/RobotTrajectory.msg`, and then click `GENERATE!`
```
Comment (mpinol) [inline tutorials/pick_and_place/2_ros_tcp.md:48]: Would it make sense to change the message generation steps to use the new MessageGenerationBrowser?

File: tutorials/pick_and_place/2_ros_tcp.md [hunk 2]
```diff
@@ -119,23 +126,27 @@ To enable communication between Unity and ROS, a TCP endpoint running as a ROS n
...
+   Select the Target object in the Hierarchy and assign it to the `Target` field in the Publisher. Similarly, assign the TargetPlacement object to the `TargetPlacement` field. Assign the niryo_one robot to the `Niryo One` field.
 
-1. Note that these components show empty member variables in the Inspector window, which need to be assigned. 
+   ![](img/2_target.gif)
 
-   Select the Target object in the Hierarchy and assign it to the `Target` field in the Publisher. Similarly, assign the TargetPlacement object to the `TargetPlacement` field. Assign the niryo_one robot to the `Niryo One` field. Finally, assign the newly created RosConnect object to the `Ros` field.
+1. Next, the ROS TCP connection needs to be defined. Select `Robotics -> ROS Settings` from the top menu bar. 
 
-   ![](img/2_target.gif)
+   In the ROS Settings window, the `Host Name` should be the IP address of your ROS machine (*not* the one running Unity).
```
Comment (LaurieCheers-unity) [inline tutorials/pick_and_place/2_ros_tcp.md:137]: The field is called "ROS IP Address"

**Review by LaurieCheers-unity — CHANGES_REQUESTED** [review]

**Review by LaurieCheers-unity — APPROVED** [review]

**Review by sarah-gibson — APPROVED** [review]
Approved, pending:
1) fix the echo ROS_IP problem in Dockerfile
2) check on the issue with not being able to pick up the cube

**Comment by at669** [comment]
> 1. fix the echo ROS_IP problem in Dockerfile

removed echo from Dockerfile, moved to `set up workspace` script. appears to be functional for me--added some troubleshooting tips just in case

> 2. check on the issue with not being able to pick up the cube

swapped gripper open/close values to match changes in Unity-Technologies/Unity-Robotics-Hub#84

**Pull request merged**